scripts/docker/examples/python/Dockerfile

```diff
@@ -0,0 +1,65 @@
+# `*:latest` will be replaced with a corresponding version stored in the '.tool-versions' file
+# hadolint ignore=DL3007
+FROM python:latest as base
+
+# === Builder ==================================================================
+
+FROM base AS builder
+ENV \
+    GOSU_VERSION="1.16"
+COPY ./assets/hello_world/requirements.txt /requirements.txt
+WORKDIR /packages
+RUN set -eux; \
+    \
+    # Get base packages
+    apk add --no-cache \
+        ca-certificates \
+        dpkg \
+        gnupg \
+        wget \
+    && \
+    \
+    # Get gosu
+    dpkgArch="$(dpkg --print-architecture | awk -F- '{ print $NF }')" && \
+    wget -O /usr/local/bin/gosu "https://github.com/tianon/gosu/releases/download/$GOSU_VERSION/gosu-$dpkgArch" && \
+    wget -O /usr/local/bin/gosu.asc "https://github.com/tianon/gosu/releases/download/$GOSU_VERSION/gosu-$dpkgArch.asc" && \
+    export GNUPGHOME="$(mktemp -d)" && \
+    gpg --batch --keyserver hkps://keys.openpgp.org --recv-keys B42F6819007F00F88E364FD4036A9C25BF357DD4 && \
+    gpg --batch --verify /usr/local/bin/gosu.asc /usr/local/bin/gosu && \
+    gpgconf --kill all && \
+    rm -rf "$GNUPGHOME" /usr/local/bin/gosu.asc && \
+    chmod +x /usr/local/bin/gosu && \
```

Comment:
Do we need this?  We're not actually using `gosu` for anything other than running `true`.  If we just want to check that we can switch users, `runit` is in the alpine repositories and would give us a `chpst` binary that's intended for the same purpose.

Reply:
It's an attempt to address an issue flagged up by SonarCloud. Now, thinking about it, this example is complex. Removed! Thanks